src/python/sortingview/gui/commonComponents/SortingUnitPairPlotGrid/SortingUnitPairPlotGrid.tsx

```diff
@@ -12,6 +12,11 @@ type Props = {
 }
 
 const SortingUnitPairPlotGrid: FunctionComponent<Props> = ({ sorting, selection, selectionDispatch, unitIds, unitPairComponent, sortingSelector }) => {
+    const maxNumUnits = 6
+    if (unitIds.length > maxNumUnits) {
+        return <span>Please select {maxNumUnits} or fewer units</span>
+    }
+    
```

Comment:
If we are going to allow select-all, then we better not allow more than 6 units to be selected for cross-correlograms.